AI/BattleAI/BattleAI.cpp

```diff
@@ -163,8 +164,21 @@ BattleAction CBattleAI::activeStack( const CStack * stack )
 				return BattleAction::makeCreatureSpellcast(stack, bestSpellcast->dest, bestSpellcast->spell->id);
 			else if(bestAttack.attack.shooting)
 				return BattleAction::makeShotAttack(stack, bestAttack.attack.defender);
-			else
-				return BattleAction::makeMeleeAttack(stack, bestAttack.attack.defender->getPosition(), bestAttack.tile);
+			else {
+/*				{

```

Comment:
Please do not commit commented out code. Define macro, as above.

Reply:
Same thing. I'm not asking to review my code, yet.

Reply:
Ahh, sorry then. Someone on SLack asked me to do that :P

Reply:
No problem, I haven't touched this code for a month, so I figured I better committed what I had sitting on my HD..